library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+        ** Posterior-dominant-rhythm-amplitude-range <nowiki> {suggestedTag=Low, suggestedTag=Medium, suggestedTag=High}[Low: less than 20 microV, Medium: 20 to 70 microVSource, High: more than 70 microV. (Source: Beniczky ea 2017, Table 4.)] </nowiki>
 
-        ** Posterior-dominant-rhythm-eye-opening-reactivity <nowiki> {suggestedTag=Property-not-possible-to-determine} [Change (disappearance or measurable decrease in amplitude) of a posterior dominant rhythm following eye-opening. Eye closure has the opposite effect.]</nowiki>
-            *** Posterior-dominant-rhythm-eye-opening-reactivity-reduced-left <nowiki>[Reduced left side reactivity.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Posterior-dominant-rhythm-eye-opening-reactivity-reduced-right <nowiki>[Reduced right side reactivity.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[free text]</nowiki>
-            *** Posterior-dominant-rhythm-eye-opening-reactivity-reduced-both <nowiki>[Reduced reactivity on both sides.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Posterior-dominant-rhythm-eye-opening-reactivity <nowiki> {suggestedTag=Property-not-possible-to-determine, suggestedTag=Left, suggestedTag=Right} [Change (disappearance or measurable decrease in amplitude) of a posterior dominant rhythm following eye-opening. Eye closure has the opposite effect. (Source: Beniczky ea 2013, Appendix S2; Beniczky ea 2017, Table 4.)]</nowiki>
 
-        ** Posterior-dominant-rhythm-organization <nowiki> {suggestedTag=Normal} [When normal could be labeled with suggested tag.]</nowiki>
-            *** Posterior-dominant-rhythm-organization-poorly-organized <nowiki>[Poorly organized.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Posterior-dominant-rhythm-organization-disorganized <nowiki>[Disorganized.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Posterior-dominant-rhythm-organization-markedly-disorganized <nowiki>[Markedly disorganized.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Posterior-dominant-rhythm-organization <nowiki> {suggestedTag=Normal} [Posterior dominant rhythm organization. When normal could be labeled with suggested tag. (Source: Beniczky ea 2017, Table 4.)]</nowiki>
+            *** Posterior-dominant-rhythm-organization-poorly-organized <nowiki>[Poorly organized. (Source: Beniczky ea 2017, Table 4.)]</nowiki>
+            *** Posterior-dominant-rhythm-organization-disorganized <nowiki>[Disorganized. (Source: Beniczky ea 2017, Table 4.)]</nowiki>
+            *** Posterior-dominant-rhythm-organization-markedly-disorganized <nowiki>[Markedly disorganized. (Source: Beniczky ea 2017, Table 4.)]</nowiki>
```

Comment:
Is there a quantification of the difference between disorganized and markedly disorganized?

Reply:
nope, seems like a 4-point scale

Reply:
thanks again for checking these!